docker/Dockerfile

```diff
@@ -1,14 +1,14 @@
 FROM nervos/ckb-riscv-gnu-toolchain:bionic-20211214
 
 # Install Rust
-RUN curl https://sh.rustup.rs -sSf | sh -s -- --default-toolchain nightly-2021-12-25 -y
+RUN curl https://sh.rustup.rs -sSf | sh -s -- --default-toolchain nightly-2022-08-01 -y
 ENV PATH=/root/.cargo/bin:$PATH
 # Install RISC-V target
 RUN rustup target add riscv64imac-unknown-none-elf
 # Install CKB binary patcher
-RUN cargo install --git https://github.com/nervosnetwork/ckb-binary-patcher.git --rev 930f0b468a8f426ebb759d9da735ebaa1e2f98ba
+RUN cargo install --git https://github.com/nervosnetwork/ckb-binary-patcher.git --rev b9489de4b3b9d59bc29bce945279bc6f28413113
```

Comment:
It's not needed: the bug introduced by old ckb-vm has been fixed.

Reply:
OK, may also mention it in `ckb-binary-patcher`'s README?